nsv13/code/modules/overmap/weapons/projectiles_fx.dm

```diff
@@ -205,15 +211,41 @@ Misc projectile types, effects, think of this as the special FX file.
 			return FALSE //Didn't take the hit
 	if(!isprojectile(target)) //This is lazy as shit but is necessary to prevent explosions triggering on the overmap when two bullets collide. Fix this shit please.
 		detonate(target)
+	else
+		return FALSE
 	return BULLET_ACT_HIT
 
+/obj/item/projectile/guided_munition/proc/spec_overmap_hit(obj/structure/overmap/target)
+	return
+
+/obj/item/projectile/guided_munition/torpedo/disruptor/spec_overmap_hit(obj/structure/overmap/target)
+	if(QDELETED(target))
+		return
+	if(target.occupying_levels.len)
```

Comment:
```suggestion
	if(length(target.occupying_levels))
```

Reply:
Is a length() call that handles multiple variations of types really more efficient than a single variable access?

Reply:
yes